Description: Add missing flags in Makefile
 Add missing LDFLAGS, CPPFLAGS and CFLAGS in Makefile and drop -Werror
Author: Sophie Brun <sophie@freexian.com>
Origin: vendor
Last-Update: 2015-04-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/Makefile
+++ b/src/Makefile
@@ -1,12 +1,12 @@
 CC = gcc
-CCFLAGS = -lssl -lcrypto -Wall -Werror
+CCFLAGS = -lssl -lcrypto -Wall
 
 TARGET = pixiewps
 PREFIX = $(DESTDIR)/usr
 BINDIR = $(PREFIX)/bin
 
 all:
-	$(CC) -o $(TARGET) pixiewps.c $(CCFLAGS)
+	$(CC) -o $(TARGET) pixiewps.c $(CCFLAGS) $(CFLAGS) $(LDFLAGS) $(CPPFLAGS)
 
 install:
 	install -D pixiewps $(BINDIR)/$(TARGET)
